### ISSUE TITLE
Widget screen: Adding widgets/blocks to widget area 2 adds them to widget area 1

### ISSUE THREAD

**Issue by paaljoachim** [WordPress/gutenberg#25764]
**Describe the bug**
I noticed that when Widget area 2 (Footer 2) is open that adding a block through the inserter panel adds it automatically to widget area 1.

**To reproduce**
1. Open widget area 2 (Footer 2) (Footer widget area one is closed)
2. Open the Inserter Panel. Click to add a block/widget.
3. Notice it is not added to Footer 2, but to Footer 1.

**Expected behavior**
I expected to add widgets/blocks to the widget area that is open. As Footer widget area 1 was closed.

![Widget-screen-Open-widget-area-2-adds-widgets-to-1](https://user-images.githubusercontent.com/5323259/94783678-50943000-03cd-11eb-8a94-5c6cf0aa4061.gif)

**Editor version (please complete the following information):**
- WordPress version: 5.5.1
- Does the website has Gutenberg plugin: 9.1

**Desktop (please complete the following information):**
 - Mac
 - Chrome
 - Desktop Server local site.

Bottom line is that I am only able to add widgets/blocks to Footer 1. I am not able to add widgets to any other widget area.

**Comment by noisysocks** [comment]
https://github.com/WordPress/gutenberg/pull/25727 fixes this.
